doc/user_guide.md

```diff
@@ -551,6 +551,165 @@ Finally duplicate links can exist. They have no real direction. the closest matc
 |--------|-------------------|
 | ?      | Duplicate         |
 
+## XML tracing report
+
+Often projects want to further process the results generated by OpenFastTrace to create statistics about requirements, 
+to add the results to a database or further analyze the results. OpenFastTrace therefore provides a reporter that
+exports all relevant collected into a single XML file that can further be processed by other tools.
+
+The XML exporter is called `aspec` reporter. An `aspec` report can be generated by calling OpenFastTrace in the 
```

Comment:
Is `aspec` an abbreviation? Maybe you can add a short explanation for the name ;)

Reply:
DONE